### PR TITLE
add start/end moving step

### PR DESCRIPTION
**Pull request by jiecaix**
Add 'warpx_start_moving_window_step' and 'warpx_end_moving_window_step' for class CylindricalGrid/Cartesian1DGrid to make python interface could stop the moving window

### CODE REVIEW

File: Python/pywarpx/picmi.py [hunk 1]
```diff
@@ -598,6 +598,9 @@ def init(self, kw):
         self.potential_zmax = kw.pop('warpx_potential_hi_z', None)
         self.reflect_all_velocities = kw.pop('warpx_reflect_all_velocities', None)
 
+        self.warpx_start_moving_window_step = kw.pop('warpx_start_moving_window_step', 0)
```
Comment (dpgrote) [inline Python/pywarpx/picmi.py]: The default values here should be `None`, as is done for the other parameters. The default values of these parameters (i.e. 0 and -1) are already set in the c++ and shouldn't be duplicated here.

**Comment by dpgrote** [comment]
Thanks for this PR! It looks good, though I have the comment on the default values. Also, can you add the two new parameters to the documentation by including them in the list of parameters in the triple quoted doc strings?

File: Python/pywarpx/picmi.py (hunk 1 above)
Comment (jiecaix) [reply Python/pywarpx/picmi.py]: Thanks for your comment, and i had change them to be None. Also, i add the description of two new parameters to the triple quoted doc strings.

File: Python/pywarpx/picmi.py [hunk 2]
```diff
@@ -581,6 +581,13 @@ class CylindricalGrid(picmistandard.PICMI_CylindricalGrid):
     warpx_reflect_all_velocities: bool default=False
         Whether the sign of all of the particle velocities are changed upon
         reflection on a boundary, or only the velocity normal to the surface
+
+    warpx_start_moving_window_step: int, default=0
+        The timestep at which the moving window starts
+
+    warpx_end_moving_window_step: int, default=-1
+        The timestep at which the moving window ends. If -1, the moving window
+    will continue until the end of the simulation.
```
Comment (dpgrote) [inline Python/pywarpx/picmi.py]: One last detail - can you fix the indentation?

Comment (jiecaix) [reply Python/pywarpx/picmi.py]: Thanks for this check, the indentation has been fixed.

**Review by dpgrote — APPROVED** [review]
Looks good! Thanks!

**Pull request merged**